test/test_jit.py

```diff
@@ -13126,10 +13126,10 @@ class TestJitGeneratedFunctional(JitTestCase):
     ('softmin', (S, S, S), (0,),),
     ('softmax', (S, S, S), (0,), '', (True,)),
     ('softmax', (S, S, S), (0, 3, torch.double), 'with_all_args', (True,)),
-    ('tanh', (S, S, S), (),),
-    ('sigmoid', (S, S, S), (),),
+    ('tanh', (S, S, S), (), '', (True,)),
+    ('sigmoid', (S, S, S), (), '', (True,)),
     ('log_softmax', (S, S, S), (0,), '', (True,)),
-    ('linear', (S, S), ((M, S),),),
+    ('linear', (S, S), ((M, S),), '', (True, ['aten::t', 'aten::matmul'])),
```

Comment:
Question: could we make checking ad on by default?

Reply:
Hmmm there're way to many ops that we don't have AD than having one. I would flip this setting when we have a better coverage in AD :D